eng/restore/repoRestore.props

```diff
@@ -18,6 +18,7 @@
 
   <!-- Enable the repo-restored analyzers -->
   <PropertyGroup>
+    <AnalyzerPropsFile>$(ToolSetCommonDirectory)Tools.Analyzers.props</AnalyzerPropsFile>
```

Comment:
where's this property used?

Reply:
This property is used in [`CodeAnalysis.targets`](https://github.com/dotnet/corefx/blob/master/eng/CodeAnalysis.targets#L7) and in the new depproj I added in order to write [the file here](https://github.com/dotnet/corefx/pull/41832/files#diff-aaf403c351cb24fc2af085a0a8df4ba5R18).